client/src/components/main/main.component.tsx

```diff
@@ -0,0 +1,27 @@
+import './main.styles.scss'
+
+import SideBarComponent from './../sidebar/sidebar.component'
+import HeaderComponent from './../header/header.component'
+import SearchBoxComponent from './../search-box/search-box.component'
```

Comment:
You may not need the first dot. `../search-box/search-box.component` may be enough.